### ISSUE TITLE
Google block ?

### ISSUE THREAD

**Issue by mikeruss1** [CIDRAM/CIDRAM#441]
this looks valid to me ?

IP address: 64.233.172.8 IP address (resolved): - 
User agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36 Google-PageRenderer Google (+https://developers.google.com/+/web/snippet/) 
Signatures count: 1 Signatures reference: bobuam.php:L128 
Why blocked: Suspected bot or scraper probe!
Hostname: google-proxy-64-233-172-8.google.com 

its a bit odd its using Chrome 56 but apparently OK?
and the page its looking for is one I recently asked Google to add to the index

**Comment by Maikuolan** [comment]
Still investigating this one. I'll reply back with more details when I get a bit further with it.

**Comment by mikeruss1** [comment]
no rush, have let it through with an aux rule

**Comment by 737simpilot** [comment]
Google proxy is an interesting one. For me, I know people can use Google proxy to hide their IP, so I flat out ban the rDNS for it. But! It comes at a cost because I believe Google translate will show up as Google proxy. For me it's not much of an issue because I have a rule on my low SEO driven website that all posts are to be made in English only. On the other hand, those that want to just read and don't understand English and need to use Google translate are blocked. At one time I did have an automatic website Google translate in the website its self, but in an effort to rid my website of all Google I got rid of it. 

The UA for developers is an interesting one.
